src/edit/tools/getEXIFdata.js

```diff
@@ -0,0 +1,85 @@
+L.EXIF = function getEXIFdata(img) {
```

Comment:
This looks great, however isn't it only displaying in the console? Can we add a returned value from this function and then:

1. move the image to that location on the planet
2. resize it, rotate it using the available data, as in the below code
3. pan the map to the new position so the user can see it

https://github.com/publiclab/mapknitter/blob/6d8d8b5b3f6a8dfd8415f3a70efbeccefc73f044/app/assets/javascripts/mapknitter/Map.js#L180-L227 

What do you think? Thanks @rexagod this is very cool! And this new way of organizing the code is awesome. Thanks a lot.

Reply:
@jywarren How's this? Committing this as well.

![Peek 2019-03-18 03-00](https://user-images.githubusercontent.com/33557095/54498211-3b11ff00-492a-11e9-8a10-790828b01e4a.gif)

Reply:
>move the image to that location on the planet
>resize it, rotate it using the available data, as in the below code
>pan the map to the new position so the user can see it

@jywarren I've implemented all of the above except the resizing (scaling/distort) one since there is no available data for that in the EXIF tags. What do you think? Thanks!